src/OpenTelemetry.Extensions.Hosting/Implementation/TelemetryHostedService.cs

```diff
@@ -36,12 +37,18 @@ public Task StartAsync(CancellationToken cancellationToken)
         {
             try
             {
-                // The sole purpose of this HostedService is to ensure
-                // all instrumentations are created and started.
-                // This method is invoked when host starts, and
-                // by requesting the TracerProvider from DI
-                // it ensures all instrumentations gets started.
-                this.serviceProvider.GetRequiredService<TracerProvider>();
+                // The sole purpose of this HostedService is to ensure all
+                // instrumentations are created and started. This method is
+                // invoked when host starts, and by requesting the
+                // TracerProvider & MeterProvider from DI it ensures all
+                // instrumentations gets started.
+                var tracerProvider = this.serviceProvider.GetService<TracerProvider>();
+                var meterProvider = this.serviceProvider.GetService<MeterProvider>();
```

Comment:
If there is no strong reason to put meter after trace, consider alphabetic order.

Reply:
Done